Add search link to Pontoon Translation Search (#324)
flodolo/documentation: src/products/firefox_desktop/firefox_l10n_faqs.md

### How can I see all translations available for a specific string?

> Pontoon doesn’t allow to see all translations for a specific string, but you can use Pontoon's [Translation Search](https://pontoon.mozilla.org/search/) for it:

Reviewer: The first part of the sentence is wrong…

Author: Wow. My bad :)

Author: I realized that the whole paragraph needed editing. The previous docs mentioned Transvision API. Since there are no API docs other than drf-spectacular, I removed the sentence for now.

### How can I see all translations available for a specific string?

> You can use Pontoon's [Translation Search](https://pontoon.mozilla.org/search/) for searching through all translations for a source string:

Reviewer: ```suggestion
You can use Pontoon’s [Translation Search](https://pontoon.mozilla.org/search/) for searching through all translations for a source string:
```

> * Search for the string (by text or include string identifiers), making sure that `Firefox` is selected as the project.

Reviewer: ```suggestion
* Search for a specific text, making sure that `Firefox` is selected as the project.
```
